src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -7858,51 +7942,68 @@ namespace InternalHeatGains {
                     state.dataHeatBal->ZoneITEq(Loop).DryBulbTAboveDeltaT = TAirIn - DBMax(EnvClass);
                     state.dataHeatBal->ZnRpt(NZ).ITEqTimeAboveDryBulbT = state.dataGlobal->TimeStepZone;
                     state.dataHeatBal->ZnRpt(NZ).ITEqTimeOutOfOperRange = state.dataGlobal->TimeStepZone;
+                    state.dataHeatBal->spaceRpt(spaceNum).ITEqTimeAboveDryBulbT = state.dataGlobal->TimeStepZone;
+                    state.dataHeatBal->spaceRpt(spaceNum).ITEqTimeOutOfOperRange = state.dataGlobal->TimeStepZone;
```

Comment:
I am getting a gut feeling here. Both the zone and space "time of something" is set here (not summed, but set). But there can be more than 1 space per zone. This is OK as long as all spaces in a zone have the same conditions. I think that is the case so this is probably OK.